### ISSUE TITLE
External showcase project: nice to have

### ISSUE THREAD

**Issue by graninas** [tweag/inline-js#26]
It would be nice to have some external sample app showing some basic use case of the library. This app will also provide an info how to install the library as dependency.

This is very useful because it gives some viable hints (and the .cabal / .stack template) for the clients how the library can be installed.

**Comment by bigpopakap** [comment]
This would be extremely useful. I'm a Haskell newbie, so I do need some hand-holding to get started. As it currently stands, I've read through a lot of documentation and looked through code, but I can't for the life of me figure out how to install this library

**Comment by TerrorJack** [comment]
Makes sense. I've updated `README` to add a subsection on how to import `inline-js` into external projects; don't have enough energy to maintain showcase projects outside the repo atm, but added this to my backlog.

**Comment by haroldcarr** [comment]
Can anyone point me to a current use that is working?

The blog article is using old APIs.   The so-called "examples" in the source are not helpful.

The tests seem to go a bit in the right direction.

But a simple COMPLETE example with cabal files, etc., would be very helpful.

**Comment by haroldcarr** [comment]
I have created a COMPLETE example.
However, it has a major hack.  Feedback on how to do this example "right" would be appreciated.
[haskell-inline-js-example](https://github.com/haroldcarr/haskell-inline-js-example)
